tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -1003,10 +1011,56 @@ public String toXmlString()
         return builder.toString();
     }
 
+    public IStatus reloadProperties() throws CoreException
```

Comment:
We don't need this method, if someone needs to reload they can call "getBuildProperties(true)" and then revalide if they need.